server/test/views/questiontypes/RadioButtonQuestionRendererTest.java

```diff
@@ -75,12 +75,12 @@ public void render_generatesIds_formatsWhitespaceAsUnderscore() {
   public void render_withExistingAnswer_checksThatOption() {
     QuestionAnswerer.answerSingleSelectQuestion(
         applicantData, question.getContextualizedPath(), 2L);
-    Tag result = renderer.render(params);
+    DivTag result = renderer.render(params);
 
     assertThat(result.render())
         .contains(
             "<input id=\"peanut_butter\" type=\"radio\""
                 + " name=\"applicant.favorite_ice_cream.selection\""
-                + " value=\"2\" checked=\"\"");
+                + " value=\"2\" checked");
```

Comment:
Updated cuz as a result of using `.isChecked` over [here](https://github.com/seattle-uat/civiform/blob/3c2e72e928c195c653dedf0efb82af36e34373ca/server/app/views/components/FieldWithLabel.java#L470) insead of the old `.attr(..)` over [here](https://github.com/seattle-uat/civiform/blob/1f74106e27bfb7f777dbb91b98f411a101dad933/server/app/views/components/FieldWithLabel.java#L328)